src/app/modules/submission/email-submission/email-submission.controller.ts

```diff
@@ -454,6 +440,7 @@ const submitEmailModeForm: ControllerHandler<
 }
 
 export const handleEmailSubmission = [
+  // todo: remove CaptchaMiddleware after extracting common components in Captcha and Turnstile
```

Comment:
/nit Does your plugins support lowercase todos? Mine (vscode) doesn't capture lowercase `todo`, only uppercase `TODO` 🤔

Reply:
Mine does but I will change it to uppercase